Client/Frontend/Browser/Tabs/TabTrayState.swift

```diff
@@ -4,4 +4,15 @@
 
 import Foundation
 
-struct TabTrayState {}
+struct TabTrayState {
+    var isPrivateMode: Bool
+    var isPrivateTabsEmpty: Bool
+    var isInactiveTabEmpty: Bool
+    var isInactiveTabsExpanded = false
+    var inactiveTabs = ["One",
```

Comment:
I guess this is just blank data for now until we pipe the real one into it?

Reply:
yes I left the TODO with the ticket where this object gets initialized